### PR TITLE
Improvements to file conventions and other documentation

### PR DESCRIPTION
**Pull request by fredrik-bakke**
- Reformulates file conventions
- Adds a template agda file
- misc. documentation improvements and small fixes

### CODE REVIEW

**Comment by EgbertRijke** [comment]
This is really nice! Thank you for doing this!

File: FILE-CONVENTIONS.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  `<https://unimath.github.io/agda-unimath/>`. This will be displayed as
+  <https://unimath.github.io/agda-unimath/>.
+- Or if you want to add a clickable link with custom text use
+  `[UniMath/agda-unimath](https://github.com/UniMath/agda-unimath)`. This will
+  be displayed as
+  [UniMath/agda-unimath](https://github.com/UniMath/agda-unimath).
+
+## Notes
+
+- For a template file see [`template.lagda.md`](template.md).
```
Comment (fredrik-bakke) [inline FILE-CONVENTIONS.md:93]: Will this link work, @jonaprieto?

Comment (jonaprieto) [reply FILE-CONVENTIONS.md:93]: [`template.lagda.md`](template.lagda.md).

Comment (jonaprieto) [reply FILE-CONVENTIONS.md:93]: Also, you must add this file to METAFILES to ensure it's available when generating the website.

Comment (fredrik-bakke) [reply FILE-CONVENTIONS.md:93]: Thanks!

**Comment by fredrik-bakke** [comment]
Are you okay with merging this PR?

**Comment by jonaprieto** [comment]
> Are you okay with me merging this PR?

No, but you already merged it.

**Comment by jonaprieto** [comment]
Let's reconsider this pattern of merging without final approval. I have seen this pattern in other merges. It's not your fault in principle. We didn't have other reviewers before and it was Egbert mostly reviewing/pushing things, so we didn't protect any merging for approval. It may be time to think about that.

**Comment by fredrik-bakke** [comment]
Apologies for the confusion with the merge, and thank you for your feedback. I was under the impression that I had support to merge this, but realize I should have waited for final approval. I think you're right in that we should reevaluate our merging process to prevent misunderstandings like this in the future.

**Pull request merged**